cblox/include/cblox/core/submap_collection.h

```diff
@@ -116,9 +132,16 @@ class SubmapCollection : public SubmapCollectionInterface {
   bool empty() const { return id_to_submap_.empty(); }
   size_t size() const { return id_to_submap_.size(); }
   size_t num_patches() const { return id_to_submap_.size(); }
+
+  // Note(alexmillane): These functions are unsafe and results in unexpected
+  // behaviour because they will crash if theres are no submaps. Remove!
```

Comment:
Maybe add a check that there is at least one submap in the collection?

Reply:
Good idea. Cheers.